utils/validators.ts

```diff
@@ -262,7 +262,7 @@ const inventoryItemModelProperties: Property[] = [
   {
     key: 'assignee',
     types: 'string|object',
-    required: true,
+    required: false,
```

Comment:
maybe pull in the branch where we switch to request/response models to reduce conflicts/tech debt, then make the base branch of the pr that branch

Reply:
good idea, working on it now